README.md

```diff
@@ -151,8 +153,14 @@ To run the test container use the following commands in the terminal:
 Build:
 docker build --target test --tag todo_app:test .  
 
-Run:
-docker run --env-file .env.test todo_app:test 
+Run integration tests:
```

Comment:
Thanks for the clear instructions here, but don't forget to change them if you do adjust the layout as per the suggestion above